code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -77,6 +78,188 @@
 				else
 					owner.nutrition += (10/difference)
 
+		if(digest_mode == DM_ITEMWEAK)
+			var/obj/item/T = pick(touchable_items)
+			if(istype(T, /obj/item))
+				if(istype(T, /obj/item) && _is_digestable(T) && !(T in items_preserved))
+					if(T in items_preserved)// Doublecheck just in case.
+						return
+					if(istype(T,/obj/item/weapon/card/id))// Mess up unprotected IDs
+						var/obj/item/weapon/card/id/ID = T
+						ID.desc = "A partially digested card that has seen better days.  Much of it's data has been destroyed."
+						ID.icon = 'icons/obj/card_vr.dmi'
+						ID.icon_state = "digested"
+						ID.access = list() // No access
+						items_preserved += ID
+						return
+					for(var/obj/item/SubItem in T)
+						if(istype(SubItem,/obj/item/weapon/reagent_containers/food/snacks))
+							var/obj/item/weapon/reagent_containers/food/snacks/SF = SubItem
+							SF.reagents.trans_to_mob(owner, (SF.reagents.total_volume * 0.3), CHEM_INGEST)
+							internal_contents -= SF
+							qdel(SF)
+						if(istype(SubItem,/obj/item/weapon/storage))
+							for(var/obj/item/SubSubItem in SubItem)
+								if(istype(SubSubItem,/obj/item/weapon/reagent_containers/food/snacks))
+									var/obj/item/weapon/reagent_containers/food/snacks/SSF = SubSubItem
+									SSF.reagents.trans_to_mob(owner, (SSF.reagents.total_volume * 0.3), CHEM_INGEST)
+									internal_contents -= SSF
+									qdel(SSF)
+					if(istype(T, /obj/item/weapon/reagent_containers/food/snacks)) // Weakgurgles still act on foodstuff. Hopefully your prey didn't load their bag with donk boxes.
+						var/obj/item/weapon/reagent_containers/food/snacks/F = T
+						F.reagents.trans_to_mob(owner, (F.reagents.total_volume * 0.3), CHEM_INGEST)
+						internal_contents -= F
+						qdel(F)
+					else
+						items_preserved += T
+						T.contaminate() // Someone got gurgled in this crap. You wouldn't wear/use it unwashed. :v
```

Comment:
"You wouldn't wear/use it unwashed"
I wouldn't put it past people.

Everything looks good codewise, though.